Cubical/Categories/Constructions/Subcategory.agda

```diff
@@ -0,0 +1,100 @@
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Constructions.Subcategory where
+-- General Subcategory (restrictions on objects and morphisms)
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Data.Sigma
+
+open import Cubical.Categories.Category
+
+private
+  variable
+    ℓC ℓC' ℓD ℓD' ℓE ℓE' ℓP ℓQ ℓR : Level
+
+module _ (C : Category ℓC ℓC') where
+  private
+    module C = Category C
+  open Category
+
+
+  record DisplayedCategory : Type (ℓ-max (ℓ-max ℓC ℓC') (ℓ-suc ℓP)) where
```

Comment:
This should go in its own file

Reply:
Moved to `DisplayedCategory.agda`. Left the `DisplayedPoset` and `Grothendieck` in the Subcategory file though